tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/openwire/interop/CompressedInteropTest.java

```diff
@@ -133,7 +133,7 @@ private void receiveStreamMessage(boolean useCore) throws Exception {
       byte[] bytesVal = new byte[originVal.length];
       streamMessage.readBytes(bytesVal);
       for (int i = 0; i < bytesVal.length; i++) {
-         assertTrue(bytesVal[i] == originVal[i]);
+         assertEquals(bytesVal[i], originVal[i]);
```

Comment:
Args are reversed, here and most of the other ones changed below (except the last one)